internal/digest/compute_test.go

```diff
@@ -0,0 +1,70 @@
+package digest_test
+
+import (
+	"testing"
+
+	"github.com/hatchet-dev/hatchet/internal/digest"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDigestValues(t *testing.T) {
+	tests := []struct {
+		name   string
+		values map[string]interface{}
+	}{
+		{
+			name:   "Empty map",
+			values: map[string]interface{}{},
+		},
+		{
+			name: "Single key-value pair",
+			values: map[string]interface{}{
+				"key": "value",
+			},
+		},
+		{
+			name: "Multiple key-value pairs",
+			values: map[string]interface{}{
+				"a": 1,
+				"b": 2,
```

Comment:
Keys in different orders should map to the same digest no? There's really no meaning in key ordering in Go maps since they're accessed randomly when iterated anyway, neither should there be meaning behind key ordering in JSON/YAML.